client/my-sites/email/email-management/email-home.jsx

```diff
@@ -82,7 +84,9 @@ class EmailManagementHome extends React.Component {
 				return this.renderContentWithHeader(
 					<EmailProvidersComparison
 						domain={ selectedDomain }
-						isGSuiteSupported={ hasGSuiteSupportedDomain( [ selectedDomain ] ) }
+						isGSuiteSupported={
+							hasGSuiteSupportedDomain( [ selectedDomain ] ) && userCanPurchaseGSuite
+						}
```

Comment:
Nite: Here and later on, does it make sense to check the local boolean first?
```suggestion
						isGSuiteSupported={
							userCanPurchaseGSuite && hasGSuiteSupportedDomain( [ selectedDomain ] ) 
						}
```

Reply:
Sure thing 👍  Reordered the conditions in 4e7e388966aa0848a18f7287310dc587c4a1b853